### PR TITLE
Feat/mobile&tablet layout

### PR DESCRIPTION
**Pull request by AlonaLook**

### CODE REVIEW

File: src/css/modules/buttons.scss [hunk 1]
```diff
@@ -79,7 +79,13 @@
       font-size: 16px;
       padding: 14px 24px 10px;
       margin: 0 auto;
+      max-width: 184px;
+
+      &:nth-child(n) {
```
Comment (PritulaRoman) [inline src/css/modules/buttons.scss]: Please, check is it a valid rule.

I believe `&:nth-child(n) {}` can be replaced with '& > {}'

File: src/css/modules/cards.scss [hunk 2]
```diff
@@ -110,22 +145,33 @@
...
-
     @media screen and (min-width: $break-small) {
       &:first-child {
         margin-left: 0;
       }
+
+      display: block;
+      margin: 0 0 16px 0;
     }
+    //@media screen and (min-width: $break-tablet) {
```
Comment (PritulaRoman) [inline src/css/modules/cards.scss]: Please, remove

File: src/css/style.scss [hunk 3]
```diff
@@ -13,7 +13,8 @@
 /*Header*/
 .box-bands {
   display: block;
-  @media screen and (min-width: $break-small) {
+
+  @media screen and (min-width: $break-tablet) {
```
Comment (PritulaRoman) [inline src/css/style.scss]: Please, fix the alignment

File: src/css/style.scss [hunk 4]
```diff
@@ -164,9 +178,17 @@
     height: 24px;
     margin-left: 23px;
 
-    @media screen and (min-width: $break-small) {
+    @media screen and (min-width: $break-tablet) {
       display: none;
     }
+
+    &--white {
```
Comment (PritulaRoman) [inline src/css/style.scss]: Please, give a more abstract name instead of `white`. Maybe `light`?

File: src/css/modules/cards.scss (hunk 2 above)
Comment (AlonaLook) [reply src/css/modules/cards.scss]: Remved

File: src/css/style.scss (hunk 4 above)
Comment (AlonaLook) [reply src/css/style.scss]: Changed.

**Pull request merged**